resources/aws/asg_stack.go

```diff
@@ -142,16 +167,19 @@ func (s *ASGStack) CreateOrFail() error {
 
 // Update updates the autoscaling group stack in Cloud Formation if one of the
 // updatable parameters has changed.
-func (s *ASGStack) Update() error {
+func (s *ASGStack) Update() (bool, error) {
```

Comment:
IIRC I got my feedback on this quite early but the style was continued and I still do not agree with it. IIRC there is even an issue to change this but so far nobody changed it. I hear having these information logged is useful for humans. Then I think the authority/service that implements certain functionality should just be configured with a logger like the resources are in the reconciliation framework and do the logging on their own. Then the information can be logged and the interface stays cleaner.